collectors/python.d.plugin/energid/README.md

```diff
@@ -0,0 +1,37 @@
+# energid
+
+A collector for [Energi Core](https://github.com/energicryptocurrency/energi) node instance monitoring.
+
+As Energi Core Gen 1 & 2 are based on the original Bitcoin code and
+supports very similar JSON RPC, there is quite high chance the module works
+with many others forks including bitcoind itself.
+
+Introduces several new charts:
+
+1. **Blockchain Index**
+ * blocks
+ * headers
+
+2. **Blockchain Difficulty**
+ * diff
+
+3. **MemPool** in MiB
+ * Max
+ * Usage
+ * TX Size
+
+4. **Secure Memory* in KiB
+ * Total
+ * Locked
+ * Used
+
+5. **Network**
+ * Connections
+
+6. **UTXO** (Unspent Transaction Output)
+ * UTXO
+ * Xfers (related transactions)
+
+Configuration is needed, disabled by default.
```

Comment:
There should probably be some example configuration listed here, even if it's just a trivial case.

Reply:
Done